client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
```

Comment:
@v18 _why_ should they not be an object? It's a valid point to raise _if there's a reason_, but did you find that something about this was not working properly on account of the object? it looks like we _are_ using the object for its indexing abilities